Composer/packages/client/src/recoilModel/dispatchers/project.ts

```diff
@@ -448,6 +486,32 @@ export const projectDispatcher = () => {
     }
   );
 
+  const migrateProjectTo = useRecoilCallback(
+    (callbackHelpers: CallbackInterface) => async (
+      oldProjectId: string,
+      name: string,
+      description: string,
+      location: string
+    ) => {
+      const { set, snapshot } = callbackHelpers;
+      try {
+        const dispatcher = await snapshot.getPromise(dispatcherState);
+        set(botOpeningState, true);
+
+        // starts the creation process and stores the jobID in state for tracking
+        const response = await migrateToV2(callbackHelpers, oldProjectId, name, description, location);
+
+        if (response.data.jobId) {
+          dispatcher.updateCreationMessage(response.data.jobId, '', '', '', '');
```

Comment:
CAn we make the strings optional parameters so that this line looks like `dispatcher.updateCreationMessage(response.data.jobId)`?

Reply:
yup